blog-csoar/2024-11-15-application-update.md

```diff
@@ -0,0 +1,45 @@
+---
+title: November 15, 2024 - Application Update
+keywords:
+  - sumo logic
+  - cloud soar
+  - automation service
+image: https://help.sumologic.com/img/sumo-square.png
+hide_table_of_contents: true
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-csoar/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+### Changes and Enhancements
+
+#### Automation Bridge
+
+We are happy to announce a new version of the [Automation Bridge](/docs/platform-services/automation-service/automation-service-bridge/) that includes security fixes in various components of the Bridge.
```

Comment:
This new version is to be released as beta. Can we capture the same in documentation update.

Reply:
Can we make it just "security fixes" instead of "security fixes in various components of the Bridge".